src/Discovery/aggMDSUtils.jsx

```diff
@@ -16,18 +16,9 @@ const retrieveCommonsInfo = async (commonsName) => {
  * getUniqueTags returns a reduced subset of unique tags for the given tags.
  *
  * @param {*} tags
- * @returns
+ * @returns array with duplicates removed
  */
-const getUniqueTags = (tags) => tags.reduce((tagsSoFar, nextTag) => {
-  if (tagsSoFar.has(nextTag.name)) {
-    return tagsSoFar;
-  }
-  tagsSoFar.set(nextTag.name, {
-    category: nextTag.category,
-    name: nextTag.name,
-  });
-  return tagsSoFar;
-}, new Map());
+const getUniqueTags = ((tags) => tags.filter((v, i, a) => a.findIndex((t) => (t.category === v.category && t.name === v.name)) === i));
```

Comment:
nitpicking: you probably can use `.uniq()` or `.uniqWith()` from `lodash` if that might be easier

but overall lgtm

Reply:
Good to know. I did not realize lodash was in package.json until now. If you and Zakir are ok with it as is, I will not change it.